### DISCUSSION TITLE
Congrats!

### DISCUSSION THREAD

**Discussion by GianniBYoung** [Serene-Arc/bulk-downloader-for-reddit#293] [General]
Just wanted to say congrats on 2.0! I've been using your program for the past year to download all of my favorites regularly, Thanks for all your hard work!!!

**Comment by aliparlakci** [comment]
Thanks a lot! It really feels awesome to hear such words from users. I didn't thought it would come to this point and be used by so many people when I first shared this project!

For v2, @Serene-Arc should be the one to get all the credit. Without her, v2 would never exists and the project would come to a halt. She reimplemented the project in way that it is more robust and maintainable.

**Reply by GianniBYoung** [reply]
In that case, thank you and @Serene-Arc! Ill keep an eye out for any issues I might be able to help with.

**Reply by aliparlakci** [reply]
We really appreciate extra help :)

**Reply by Serene-Arc** [reply]
Thanks for the kind words :) Hopefully my work stands the test of time lol. It's easy to say it's more maintainable when it's just me maintaining it. We'll see what other people think as the project grows!
